src/client/aureliaCLICommands.ts

```diff
@@ -12,6 +12,7 @@ export default class AureliaCliCommands {
       vscode.commands.registerCommand('extension.auGenerate', () => this.auGenerate(outputChannel)),
       vscode.commands.registerCommand('extension.auTest', () => this.runCommand(['test'], outputChannel, false)),
       vscode.commands.registerCommand('extension.auBuild', () => this.auBuild(outputChannel)));
```

Comment:
I looked over the helloworld vscode extension example/docs and noticed I had forgot to expose the command in the package.json. I've tested with F5 and it's there now.